README.md

```diff
@@ -234,14 +236,20 @@ If you already use other parser (e.g. `"parser": "babel-eslint"`), please move i
 
 The `vue-eslint-parser` uses the parser which is set by `parserOptions.parser` to parse scripts.
 
-### Can my javascript code have increased indentation?
+### Why don't work on `.vue` file?
 
-It depends on the version of eslint you're using.
+1. Make sure you don't have `eslint-plugin-html` in your config. The `eslint-plugin-html` extract the inside of `<script>` tags from the source code, but `eslint-vue-plugin` requires `<script>` tags and `<template>` tags in order to find script and template.
 
-[indent](https://eslint.org/docs/rules/indent) rule in `eslint@3.x` makes it possible, but if you use `eslint@4.x` be aware that this rule has been rewritten and is more strict now, thus it doesn't allow to have increased initial indentation.
+  ```diff
+    "plugins": [
+      "vue",
+  -   "html"
+    ]
+  ```
 
-You can however use [indent-legacy](https://eslint.org/docs/rules/indent-legacy) rule instead.
-More informations [here](https://eslint.org/docs/user-guide/migrating-to-4.0.0#indent-rewrite).
+2. Make sure your tool does linting for `.vue` files.
+  - CLI targets only `.js` files by default. You have to specify additional extensions by `--ext` option or glob patterns. E.g. `eslint "src/**/*.{js,vue}"` or `eslint src --ext .vue`.
+  - VSCode targets only JavaScript or HTML files by default. You have to add `{"autoFix": true, "language": "vue"}` into `eslint.validate` entry.
```

Comment:
`autoFix` is not necessary I guess :)
It's enought to just add another language to array, like so:
```
"eslint.validate": [
        "javascript",
        "javascriptreact",
        "vue",
        "vue-html"
    ],
```

Reply:
It's necessary because `vscode-eslint` does not autofix on additional file types by default. (https://github.com/Microsoft/vscode-eslint/issues/185#issuecomment-267570350)